cabal.project

```diff
@@ -81,6 +82,13 @@ source-repository-package
   -- https://github.com/tibbe/ekg-json/pull/12
 -- END DELETE
 
+if (impl(ghc >= 9.2))
+  source-repository-package
+    type:git
+    location: https://github.com/wavewave/apply-refact
+    tag: 2f346adb54ae3d911c9055b48fb62eb4ceab583a
+    -- https://github.com/wavewave/apply-refact/pull/1
```

Comment:
Is there a plan for upstreaming this to https://github.com/mpickering/apply-refact ?

Reply:
yup. I contacted the maintainers and kept discussing about this directly last few days. Will remove this as the new version is released. thanks.